domains/auth/Auth.API/API/Services/CryptographyService.cs

```diff
@@ -0,0 +1,56 @@
+using API.Models;
+using API.Helpers;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace API.Services;
+public class CryptographyService : ICryptographyService
+{
+    string key = Configuration.GetSecretKey();
+
+    public string EncryptState(AuthState state)
+    {
+        byte[] iv = new byte[16];
```

Comment:
IV should not be an empty array, when a new Aes is created, and random IV is automatically generated, often one would take the 16 bytes from IV and use as the first 16 bytes of the data package.

This could be done on line 20 by writing iv to the memory stream before the cryptostream.